dbt_subprojects/tokens/models/prices_v2/minute/prices_v2_dex_minute.sql

```diff
@@ -0,0 +1,11 @@
+{{ config(
+    schema='prices_v2'
+    , alias = 'dex_minute'
+    , materialized = 'view'
+    )
+}}
+
+SELECT
+*
+FROM {{ ref('prices_v2_dex_minute_raw') }}
+INNER JOIN {{ ref('prices_v2_dex_filter') }} using (blockchain, contract_address)
```

Comment:
Can't we do this filtering below in `prices_v2_dex_minute_raw` directly instead?

Reply:
opted to keep `prices_v2_dex_minute_raw` a clean union model